gpcontrib/gp_orca/gp_orca.control

```diff
@@ -0,0 +1,6 @@
+# gp_orca extension
+
+comment = 'ORCA planner'
+default_version = '1.0'
+module_pathname = '$libdir/gp_orca'
+relocatable = false
```

Comment:
If the SQL interface "create extension gp_orca" is not expected, then this file is not needed.